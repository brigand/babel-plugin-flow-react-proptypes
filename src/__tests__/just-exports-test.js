const babel = require('babel-core');
const content = `
export appcache from "./validators/appcache.js"
export assets from "./validators/assets.js"
export baseUrl from "./validators/baseUrl.js"
export production from "./validators/production.js"
`;

it('just-exports', () => {
  const res = babel.transform(content, {
    babelrc: false,
    presets: ['es2015', 'stage-1', 'react'],
    plugins: ['syntax-flow', require('../')],
  }).code;
  expect(res).toMatchSnapshot();
});
