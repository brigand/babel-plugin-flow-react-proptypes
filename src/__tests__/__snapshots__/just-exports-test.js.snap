// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`just-exports 1`] = `
"\\"use strict\\";

Object.defineProperty(exports, \\"__esModule\\", {
  value: true
});
Object.defineProperty(exports, \\"appcache\\", {
  enumerable: true,
  get: function get() {
    return _appcache2[\\"default\\"];
  }
});
Object.defineProperty(exports, \\"assets\\", {
  enumerable: true,
  get: function get() {
    return _assets2[\\"default\\"];
  }
});
Object.defineProperty(exports, \\"baseUrl\\", {
  enumerable: true,
  get: function get() {
    return _baseUrl2[\\"default\\"];
  }
});
Object.defineProperty(exports, \\"production\\", {
  enumerable: true,
  get: function get() {
    return _production2[\\"default\\"];
  }
});

var _appcache2 = _interopRequireDefault(require(\\"./validators/appcache.js\\"));

var _assets2 = _interopRequireDefault(require(\\"./validators/assets.js\\"));

var _baseUrl2 = _interopRequireDefault(require(\\"./validators/baseUrl.js\\"));

var _production2 = _interopRequireDefault(require(\\"./validators/production.js\\"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { \\"default\\": obj }; }"
`;
